rclcpp/include/rclcpp/executor.hpp

```diff
@@ -107,11 +109,13 @@ class Executor
     }
   }
 
-  void spin_node_once(rclcpp::node::Node::SharedPtr & node, bool nonblocking = false)
+  template<typename Rep = int64_t, typename Period = std::milli>
```

Comment:
I think it would be nice to do so, but as I was just talking with @jacquelinekay off-line I suggested templating it on a single typename, like `DurationT`.